package.json

```diff
@@ -49,5 +49,10 @@
     "jest": "^29.0.0",
     "ts-jest": "^29.0.2",
     "typescript": "^4.8.3"
+  },
+  "resolutions": {
```

Comment:
what are the resolutions for?

Reply:
It's to resolve the vulnerabilities in the dependabot alerts that we get for the SDK.